app/config/views.yml

```diff
@@ -264,11 +264,7 @@ ezpublish:
                             Identifier\ContentType: [place, article]
 
                 block_item_hero:
-                    image:
-                        template: "@ezdesign/block_item_wide/image.html.twig"
-                        match:
-                            Identifier\ContentType: [place]
                     default:
-                        template: "@ezdesign/default/default.html.twig"
+                        template: "@ezdesign/block_item_wide/image.html.twig"
```

Comment:
Naming is always difficult.
I am not a big fan of `block_item_wide`. I would suggest to reuse the name of the block `block_item_hero`.
I think it will be easier to understand for a newcomer.